app/utils/pool-utils.ts

```diff
@@ -14,10 +14,12 @@ const iconMapping = {
     "SLES": Icons.openSUSE,
     "SLES-HPC": Icons.openSUSE,
     "standard-data-science-vm": Icons.windows,
+    "rendering-windows2016": Icons.windows,
```

Comment:
I know this is not part of this pull request but I would move the rendering images to another tab so we have VMs, Data science Vms, and rendering vms